src/glossary.rst

```diff
@@ -4135,6 +4135,14 @@ A :dt:`receiver operand` is an :t:`operand` that denotes the :t:`value` whose
 :dp:`fls_4rme1x6romeg`
 See :s:`ReceiverOperand`.
 
+.. _fls_nG6ikjLsCW7m:
+
+record enum variant
+^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_NWyvPQmOIjo2`
+A :dt:`record enum variant` is a :t:`variant` with named :t:`[field]s` of an :t:`enum type`.
```

Comment:
I would word this as

A record enum variant is an enum variant with named fields.

since we can leverage `enum variant`.